src/index.html

```diff
@@ -8,9 +8,34 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="main">
+    <form
+      action="#"
+      class="form"
+      data-qa="big"
+    >
```

Comment:
<img width="837" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/49764881/76dad1b7-2f0c-4fc0-88b6-873026a45875">

You have only `action`